Jenkinsfile

```diff
@@ -9,67 +9,85 @@ pipeline {
         SLACK_TOKEN = credentials('slack-token')
     }
     options {
-      timeout(time: 1, unit: 'HOURS') 
+      timeout(time: 1, unit: 'HOURS')
     }
 
     stages {
-        stage('Check environment') {
-            steps {
+        stage('Test assisted-service') {
+            parallel {
+                stage('Test on k8s') {
+
+                    steps {
+
+                        check_if_minikube_is_running()
+
+                        clear_slave()
+
+                        sh '''export PATH=$PATH:/usr/local/go/bin; make build-image build-assisted-iso-generator-image'''
+                        withCredentials([usernamePassword(credentialsId: 'dockerio_cred', passwordVariable: 'PASS', usernameVariable: 'USER')]) {
+                            sh '''docker login docker.io -u $USER -p $PASS'''
+                        }
+                        sh '''export PATH=$PATH:/usr/local/go/bin; make jenkins-deploy-for-subsystem'''
+                        sh '''kubectl get pods -A'''
+                        sh '''export PATH=$PATH:/usr/local/go/bin;make subsystem-run'''
+                        sh 'make clear-deployment'
```

Comment:
You will not be able to get any log after clear-deployment.

Reply:
True